src/neuroconv/datainterfaces/ecephys/tdt/tdtdatainterface.py

```diff
@@ -0,0 +1,40 @@
+"""Author: Heberto Mayorquin"""
+from typing import Optional
+from ..baserecordingextractorinterface import BaseRecordingExtractorInterface
+from spikeinterface.extractors import TdtRecordingExtractor
+
+from ....utils.types import FolderPathType
+
+
+class TdtRecordingInterface(BaseRecordingExtractorInterface):
+    """Primary data interface class for converting Tucker-Davis Technologies (TDT) data."""
+
+    RX = TdtRecordingExtractor
+
+    def __init__(self, folder_path: FolderPathType, stream_id: Optional[str] = None, verbose: bool = True):
+        """
+        Load and prepare data for TDT
+
+        Parameters
+        ----------
+        folder_path: str or Path
+            Path to the TDT file
+        stream_id: str, optional
+            Select from multiple streams.
+        verbose: bool, True by default
+            Allows verbose.
+        """
+
+        if stream_id is None:
+            stream_id = "0"  # LFP for gin data. Other streams seem non-electrical.
```

Comment:
Why aren't you setting this as a default value in the init?

Reply:
That indeed would be better. It slipped my mind.